builder/openstack/step_run_source_server.go

```diff
@@ -57,6 +58,7 @@ func (s *StepRunSourceServer) Run(state multistep.StateBag) multistep.StepAction
 		CreateOptsBuilder: servers.CreateOpts{
 			Name:             s.Name,
 			ImageRef:         s.SourceImage,
+			ImageName:        s.SourceImageName,
```

Comment:
Just skimming through the [openstack docs](http://developer.openstack.org/api-ref-compute-v2-ext.html) and I don't see imageName anywhere. Do you have a link to where this is defined?

I'm curious what happens here if these are both defined but one is always nil. Is that expected by the API?